### ISSUE TITLE
Responsive toggle - this shouldn't be an option for a "mobile first" theme

### ISSUE THREAD

**Issue by bassettsj** [drupalprojects/kalatheme#36]
Adding the settings field to add  the meta tag is silly, this should just be on. Bootstrap has gotten rid of the "responsive" stylesheet, so we should follow their lead here.

``` html
<meta content="width=device-width" name="viewport">
```

**Comment by bassettsj** [comment]
@pirog Are you strongly advocating for this option?

**Comment by bassettsj** [comment]
@carsonblack Do you have an opinion here? I just think it is silly in this day of age to have a non-responsive theme at all...

**Comment by carsonblack** [comment]
My expectation that this is going to be responsive out of the box and is one of the main reasons why I would even use this in the first place. The settings field seems like cruft.

**Comment by bassettsj** [comment]
It is gone then in 7.x-4.x :+1:

**Comment by pirog** [comment]
That toggle doesn't change whether the theme is responsive or not. It just
hides the responsive toggling choices in kalacustomize which might be a
nice option.

On Friday, June 20, 2014, Steven Bassett notifications@github.com wrote:

> It is gone then in 7.x-4.x [image: :+1:]
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/drupalprojects/kalatheme/issues/36#issuecomment-46715449
> .

## 

Cheers,

Mike Pirog
Kalamuna
www.kalamuna.com

**Comment by bassettsj** [comment]
@pirog  it also outputs that meta tag :point_up:  that tells the browser to make the page match the viewport of the browser.

**Comment by pirog** [comment]
@bassettsj weird... it should definitely not do that. 

hiding the toggling options make sense but the theme should just be responsive, that shouldn't be a choice because duh.

**Comment by bassettsj** [comment]
Cool so we all agree.

**Comment by pirog** [comment]
Yes although looking at the code for 3.x it doesnt look like adding or removing this meta content is dependent on the responsive toggle setting at all. 

```
/**
 * Load Kalatheme dependencies.
 *
 * Implements template_preprocess_html().
 */
function kalatheme_preprocess_html(&$variables) {
  // Load all dependencies.
  _kalatheme_load_dependencies();
  // Add variables for path to theme.
  $variables['base_path'] = base_path();
  $variables['path_to_kalatheme'] = dirname(__FILE__);
  // Add meta for Bootstrap Responsive.
  // <meta name="viewport" content="width=device-width, initial-scale=1.0">
  $element = array(
    '#tag' => 'meta',
    '#attributes' => array(
      'name' => 'viewport',
      'content' => 'width=device-width, initial-scale=1.0',
    ),
  );
  drupal_add_html_head($element, 'bootstrap_responsive');
}
```
